docs/feature_rgblight.md

```diff
@@ -187,6 +187,8 @@ const uint8_t RGBLED_GRADIENT_RANGES[] PROGMEM = {255, 170, 127, 85, 64};
 
 ## Lighting Layers
 
+**Note:** Lighting Layers is an RGB Backlight feature, it will not work for RGB Matrix implementations without custom code.
```

Comment:
I would suggest this: 
```suggestion
?> **Note:** Lighting Layers is an RGB Light feature, it will not work for RGB Matrix. See [RGB Matrix Indicators](feature_rgb_matrix.md?indicators) for details on how to do so.
```